src/pages/orders/Order.tsx

```diff
@@ -86,21 +94,23 @@ export const Orders = (props: OrderProps): ReactElement => {
             checkedChildren={t('', 'Opening')}
             unCheckedChildren={t('', 'Closed')}
             checked={value === 'opening'}
-            loading={isStatusLoading?.status && isStatusLoading.id === record._id}
+            loading={statusLoading?.status && statusLoading.id === record._id}
+            onClick={(_, e: React.MouseEvent) => {
+              e.stopPropagation();
+            }}
             onChange={async (isOpen) =>
-              await dispatch?.handleOnChangeOrderStatus({ _id: record._id, status: isOpen ? 'opening' : 'closed' })
+              await handleUpdateOrderStatus({ _id: record._id, status: isOpen ? 'opening' : 'closed' })
             }
           />
         );
       },
     },
   ];
-  const dataSource: DataType[] | undefined = useMemo(
+  const dataSource: IOrderGroup[] | undefined = useMemo(
```

Comment:
@phanvohieunghia  There seems to be confusion regarding **Order** and **Operational setting.**